qdrant-landing/content/blog/qdrant-1.10.x.md

```diff
@@ -0,0 +1,204 @@
+---
+title: "Qdrant 1.10 - Universal Query, BM42 & ColBERT Support"
+draft: false
+short_description: "Single search API. New search algorithm. Native multivector support."
+description: "Consolidated search API, brand new BM42 algorithm, and native multivector support. " 
+preview_image: /blog/qdrant-1.10.x/social_preview.png
+social_preview_image: /blog/qdrant-1.10.x/social_preview.png
+date: 2024-06-25T00:00:00-08:00
+author: David Myriel
+featured: false 
+tags:
+  - vector search
+  - ColBERT late interaction
+  - BM25 algorithm
+  - search API
+  - new features
+---
+
+[Qdrant 1.10.0 is out!](https://github.com/qdrant/qdrant/releases/tag/v1.10.0) This version introduces at some major changes, so let's dive right in:
+
+**Universal Query API:** All search APIs are now consolidated into the single Query API endpoint.</br>
+**Hybrid Search:** Hybrid Search is offered out of the box, and you can use it via Query API.</br>
+**BM42 Algorithm:** Alternative method of scoring is best suited for Hybrid Search on short docs.</br>
+**Multivector Support:** Native support for late interaction ColBERT is accessible via Query API.
+
+## One Endpoint for All Queries
+**Query API** will consolidate all search APIs into a single request. You can now configure a single request to `nearest`/`fusion` search or `discover`/`recommend` or `context`/`order_by` results. Previously, you had to work outside of the API to combine different search requests. Now these approaches are reduced to parameters of a single request, so you can avoid merging individual results.
```

Comment:
> `discover`/`recommend` or `context`/`order_by` results

I don't get what these combinations mean.

If it is about re-scoring, then it should be on the hybrid section. For this "One Endpoint for All Queries" title I would expect to list the kinds of queries that this can make, independently of the capabilities of `prefetch`